docs/source/tutorials_source/package/tutorial_custom_augmentations.py

```diff
@@ -371,6 +431,7 @@ def plot_knn_multilabels(embeddings, multilabels, samples_idx, n_neighbors=50):
         bars1 = multilabels[idx]
```

Comment:
line 367/427: The samples are not picked randomly, they are hardcoded.

Reply:
Thanks, changed it.